noir-projects/noir-contracts/contracts/app_subscription_contract/src/main.nr

```diff
@@ -44,13 +44,6 @@ contract AppSubscription {
         note.remaining_txs -= 1;
         storage.subscriptions.at(user_address).replace(&mut note, true);
 
-        context.call_public_function(
-            storage.gas_token_address.read_private(),
-            FunctionSelector::from_signature("check_balance(Field)"),
-            // max gas limit in Gas tokens
-            [42]
-        );
```

Comment:
This stopped being necessary since #5267 but I forgot to remove the check from this contract.